ui.h

```diff
@@ -665,6 +665,8 @@ _UI_EXTERN int uiGridPadded(uiGrid *g);
 _UI_EXTERN void uiGridSetPadded(uiGrid *g, int padded);
 _UI_EXTERN uiGrid *uiNewGrid(void);
 
+_UI_EXTERN int uiOpenURL(const char *url);
```

Comment:
I would add the following comments:

```c
// TODO figure out how every platform behaves if the URL is not a proper URL with method (that is, "www.google.com" instead of "http://www.google.com")
```